monitoring/v3/README.md

```diff
@@ -5,22 +5,25 @@ Monitoring API to retrieve and modify Alerts data.
 
 ## Prerequisites to run locally:
 
-    * [Maven 3](https://maven.apache.org)
-    * [GCloud CLI](https://cloud.google.com/sdk/gcloud/)
-    * Create a Cloud project
+* [Maven 3](https://maven.apache.org)
+* [GCloud CLI](https://cloud.google.com/sdk/gcloud/)
+* Create a Cloud project
 
 # Set Up Your Local Dev Environment
 
 Create local credentials by running the following command and following the oauth2 flow:
 
-    gcloud beta auth application-default login
+```bash
+gcloud auth application-default login
+```
 
 To run:
 
-    * `mvn clean install`
-    * `./manage_alerts_sample.sh <YOUR-PROJECT-ID> <command> <args>
-
-## Running on GCE, GAE, or other environments
+```bash
+mvn clean install  
+./manage_alerts_sample.sh "<command> <args>"
```

Comment:
Should we have a list of commands and potential args somewhere?

Reply:
Will add this